PsfLauncher/PsfPowershellScriptRunner.h

```diff
@@ -269,6 +269,10 @@ class PsfPowershellScriptRunner
 					SSWrapper = file.path();
 				}
 			}
+			if (!std::filesystem::exists(SSWrapper))
+			{
+				MessageBoxEx(NULL, L"'StartingScriptWrapper.ps1' was not found. This is required to run the PowerShell scripts through PSF. We recommend you to add it under the package root folder and re-install the application.", L"MSIX Packet Support Framework", MB_OK | MB_ICONWARNING, 0);

```

Comment:
NIT: instead of saying "Package root" put the file path, if possible, or, at least, a path inside the package.  No need to have the developer guess if we can tell them where to put it.

Reply:
After discussion with PMs, the following display messages are suggested: 

**For StartingScriptWrapper.ps1:** 
'StartingScriptWrapper.ps1' was not found. This is required to run the PowerShell scripts through PSF. Kindly add it under the package by downloading it again from the master release of PSF and then re-install the application. 

**And for Config.json:**
‘Config.json’ was not found. Kindly add it in the package and re-install the application.

Let me know you opinion on this.

Reply:
> Kindly add it in the package and re-install the application.

Can config.json be added anywhere, or in a specific spot?

Reply:
As per our understanding PSF looks for config file in full package, hence we added comment "'Config.json' was not found. Kindly add it in package and re-install the application."

https://github.com/microsoft/MSIX-PackageSupportFramework/blob/master/PsfRuntime/Config.cpp#L265